src/styles/MuiThemeProvider.js

```diff
@@ -24,7 +28,7 @@ export const MUI_SHEET_ORDER = [
 
   'Popover',
   'Dialog',
-  'DialogAction',
+  'DialogActions',
```

Comment:
Wow, the regression test suite is really saving our ass here 😲 👏  @nathanmarks.
That Dialog block should most likely be after `Button`.
Might be a good idea to prioritize having a good visual regressions coverage as contribution are more frequent :).